scripts/utils.ts

```diff
@@ -350,7 +350,15 @@ export async function checkInstanceHealth(instanceId: string, region: string = "
         InstanceIds: [instanceId],
         DocumentName: "AWS-RunShellScript",
         Parameters: {
-          commands: ["screen -ls"]
+          commands: [
+            "source ~/.env && " +
+            "if [ \"$AGENT_ENVIRONMENT\" = \"preview\" ] && [ -n \"$PR_NUMBER\" ]; then " +
```

Comment:
This if/else logic should happen at this outer layer before we send the command to the ec2, not within the ec2.

Also no need for the source .env line, scripts will do that already. The source command is not found anyway

Reply:
Like the interpolation should happen with the version number of the package, we just confirmed with the ls. Which is great bc we don't care about pr or main instance